src/python/pants/engine/internals/parser_test.py

```diff
@@ -17,18 +18,70 @@ def test_imports_banned() -> None:
     assert "Import used in dir/BUILD at line 4" in str(exc.value)
 
 
-def test_unrecognized_symbol() -> None:
-    parser = Parser(
-        target_type_aliases=["tgt"],
-        object_aliases=BuildFileAliases(
-            objects={"obj": 0},
-            context_aware_object_factories={"caof": lambda parse_context: lambda _: None},
-        ),
+def test_unrecogonized_symbol() -> None:
+    def perform_test(tta: list, good_err: str) -> None:
+        parser = Parser(
+            target_type_aliases=tta,
```

Comment:
Note that you register `tgt` every time in your 4 cases. Instead, you can deduplicate a bit by changing `tta` to be `extra_targets: list[str]`. Then, merge the two like this:

```python
target_type_aliases=["tgt", *extra_targets]
```

This will "unpack" the `extra_targets` list.